depin/services/roles/hychain/files/hychain.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=hychain service
+After=network.target 
+ 
+[Service]
+User=root
+Group=root
+Type=simple
+ExecStart=/bin/bash /home/hychain/cli/linux/hychain-start.sh
+Restart=always
+RestartSec=5
+PIDFile=/tmp/hychain.pid
+ 
```

Comment:
RuntimeMaxSec=6h

> is required so that the script is restarted via systemd. This probably would allow us to remove `--loop-internal-ms` but I haven't tested it yet. We could also leave it and have it as a failsafe